pkg/infra/slugify/slugify.go

```diff
@@ -52,7 +56,9 @@ var (
 func Slugify(value string) string {
 	s := simpleSlugger.Slugify(strings.TrimSpace(value))
 	if len(s) > 50 || s == "" {
-		s = uuid.NewSHA1(uuid.NameSpaceOID, []byte(value)).String()
+		h := sha1.New()
+		h.Write([]byte(s))
+		s = hex.EncodeToString(h.Sum(nil))[:7]
```

Comment:
Trying to recall whether we need to worry about backward compatibility any more, I think these are just used to make friendly urls now but it's been a while.

Reply:
>  I think these are just used to make friendly urls

yup -- they are not stored, but the frontend will redirect if the slug does not match what is returned from a query.  2 years ago 🤯  we replaced [gosimple.Slugify](https://github.com/gosimple/slug) with an [embedded version ](https://github.com/grafana/grafana/pull/59517)without the 1MB of UTF data.  The change will add redirects when the slug does not match, but the slug alone can not be used to search for dashboards